### PR TITLE
Update traces.md

### PR DESCRIPTION
**Pull request by melbgirl**
Preview: https://deploy-preview-1562--opentelemetry.netlify.app/docs/concepts/signals/traces/

### CODE REVIEW

**Review by cartermp — COMMENTED** [review]
Thanks @melbgirl! Just some minor changes but otherwise I like the updates to this page.

**Review by chalin — COMMENTED** [review]
Thanks for the review @cartermp.
@melbgirl - once you're done addressing @cartermp's feedback, I'll run the markdown prettifier before merging (let me know if you'd like to try it yourself -- you'll need to work from the command line for that).

File: content/en/docs/concepts/signals/traces.md [hunk 1]
```diff
@@ -90,106 +93,111 @@ Sample Trace:
     ],
 }
 ```
-
-To understand how tracing in OpenTelemetry works, let's look at a list of
-components that will play a part in instrumenting our code:
-
+ 
+A key Trace component in OpenTelemetry is a Span:
```
[the remark this answers is not in the document]
Comment (chalin) [reply content/en/docs/concepts/signals/traces.md:97]: Here and elsewhere: don't capitalize "trace", nor "span".

[the remark this answers is not in the document]
Comment (chalin) [reply content/en/docs/concepts/signals/traces.md:97]: End the phrase with a period.

File: content/en/docs/concepts/signals/traces.md [hunk 2]
```diff
@@ -90,106 +93,111 @@ Sample Trace:
...
-
+ 
 ### Tracer
-
-A Tracer creates spans containing more information about what is happening for a
-given operation, such as a request in a service. Tracers are created from Tracer
-Providers. In some languages, a global Tracer is already initialized for you.
-
+ 
+A `Tracer` creates spans, where each span has information about what's happening
```
Comment (chalin) [inline content/en/docs/concepts/signals/traces.md:113]:
```suggestion
A **tracer** creates spans, where each span has information about what's happening
```

File: content/en/docs/concepts/signals/traces.md [hunk 3]
```diff
@@ -90,106 +93,111 @@ Sample Trace:
...
+for a given operation; for example a request in a service. Tracers are created
+from `TracerProvider`s. In some languages, a global Tracer is already
```
Comment (chalin) [inline content/en/docs/concepts/signals/traces.md:115]:
```suggestion
by a **tracer provider**. In some languages, a global tracer is already
```

File: content/en/docs/concepts/signals/traces.md [hunk 4]
```diff
@@ -90,106 +93,111 @@ Sample Trace:
...
+initialized for you.
+ 
+### Tracer Provider
+ 
+A **Tracer Provider** (sometimes called `TracerProvider`) is a factory to create
+Tracers. In most applications, a Tracer Provider is initialized once, and the
+Trace Provider's lifecycle matches the application's lifecycle.
```
Comment (chalin) [inline content/en/docs/concepts/signals/traces.md:122]:
```suggestion
A **tracer provider** is a factory used to create tracers. 
In most applications, a tracer provider is initialized once, and the
tracer provider's lifecycle matches the application's lifecycle.
```

File: content/en/docs/concepts/signals/traces.md [hunk 5]
```diff
@@ -90,106 +93,111 @@ Sample Trace:
...
+ 
+Initializing a Tracer Provider is typically the first step in tracing with
+OpenTelemetry. The Tracer Provider's initialization also includes Resource and
+Exporter initialization. In some language SDKs, a global Tracer Provider is
+already initialized for you.
```
Comment (chalin) [inline content/en/docs/concepts/signals/traces.md:127]: Fix capitalized in this paragraph.

File: content/en/docs/concepts/signals/traces.md [hunk 6]
```diff
@@ -90,106 +93,111 @@ Sample Trace:
...
+ 
 ### Trace Exporters
-
-Trace Exporters send traces to a consumer. This consumer can be standard output
-for debugging and development-time, the OpenTelemetry Collector, or any
-open-source or vendor backend of your choice.
-
+ 
+**Trace Exporters** send traces to a consumer. The consumer can be standard output
+(std.out) for debugging and development-time, the OpenTelemetry Collector, or
```
Comment (chalin) [inline content/en/docs/concepts/signals/traces.md:132]:
```suggestion
A **trace exporter** sends traces to a consumer. The consumer can be standard output
for debugging and development-time, the OpenTelemetry Collector, or
```

File: content/en/docs/concepts/signals/traces.md [hunk 7]
```diff
@@ -90,106 +93,111 @@ Sample Trace:
...
 ### Trace Context
-
-Trace Context is metadata about trace spans that provides correlation between
-spans across service and process boundaries. For example, let's say that Service
-A calls Service B and you want to track the call in a trace. In that case,
-OpenTelemetry will use Trace Context to capture the ID of the trace and current
-span from Service A, so that spans created in Service B can connect and add to
+ 
+The **Trace Context** is metadata about trace **Spans**. It provides correlation between
+Spans, across service and process boundaries. For example, let's say that
```
Comment (chalin) [inline content/en/docs/concepts/signals/traces.md:138]:
```suggestion
A **trace context** is data about trace spans. It provides correlation between
spans, across service and process boundaries. For example, let's say that
```

Comment (chalin) [reply content/en/docs/concepts/signals/traces.md:138]: @cartermp @melbgirl et al.: note the suggested change of "metadata" to just "data". Does that make sense to you?

File: content/en/docs/concepts/signals/traces.md [hunk 8]
```diff
@@ -90,106 +93,111 @@ Sample Trace:
...
+ServiceA calls ServiceB, and you want to track that call in a trace. In this case,
+OpenTelemetry will use Trace Context to capture the trace ID and current
+span from ServiceA, so that Spans created in ServiceB can connect and add to
 the trace.
-
-This is known as Context Propagation.
-
+ 
+This is known as **Context Propagation**.
```
Comment (chalin) [inline content/en/docs/concepts/signals/traces.md:144]:
```suggestion
This is known as **context propagation**.
```

File: content/en/docs/concepts/signals/traces.md [hunk 9]
```diff
@@ -90,106 +93,111 @@ Sample Trace:
...
+ 
 ### Context Propagation
-
-Context Propagation is the core concept that enables Distributed Tracing. With
+ 
+**Context Propagation** is the core concept that enables Distributed Tracing. With
```
Comment (chalin) [inline content/en/docs/concepts/signals/traces.md:148]:
```suggestion
**Context propagation** is the core concept that enables distributed tracing. With
```

**Review by chalin — COMMENTED** [review]
One more general comment: terms (like span or context) shouldn't be capitalized unless they're used at the start of a sentence.

@melbgirl - could you fix these throughout the changes you've made? I marked a few below via single suggested changes, but I didn't add a note to all of them.

File: content/en/docs/concepts/signals/traces.md (hunk 7 above)
Comment (cartermp) [reply content/en/docs/concepts/signals/traces.md:138]: That makes sense to me.

File: content/en/docs/concepts/signals/traces.md (hunk 4 above)
Comment (melbgirl) [reply content/en/docs/concepts/signals/traces.md:122]: How do we distinguish the descriptive words (eg "tracer provider") vs the actual object name?

File: content/en/docs/concepts/signals/traces.md (hunk 7 above)
Comment (melbgirl) [reply content/en/docs/concepts/signals/traces.md:138]: Yep, makes sense.

**Comment by melbgirl** [comment]
I'd love to also include a sample of a linked span (to go in the Spans section). I wasn't able to create an example from the docs alone. @cartermp or @chalin could either of you suggest one?

**Comment by cartermp** [comment]
@melbgirl here's an example from a sample app (you can probably just copy-paste the structure and plug in fields that make more sense w.r.t. the example in docs:

```json
{
    "name": "do_roll",
    "context": {
        "trace_id": "0xcebe72a8222da32d558132e0cd0546d0",
        "span_id": "0x1a42a0ed801c26da",
        "trace_state": "[]"
    },
    "kind": "SpanKind.INTERNAL",
    "parent_id": "0x7c0657cf75ba73f1",
    "start_time": "2022-08-05T14:51:38.065628Z",
    "end_time": "2022-08-05T14:51:38.065666Z",
    "status": {
        "status_code": "UNSET"
    },
    "attributes": {
        "roll.value": 2
    },
    "events": [],
    "links": [
        {
            "context": {
                "trace_id": "0xcebe72a8222da32d558132e0cd0546d0",
                "span_id": "0x7c0657cf75ba73f1",
                "trace_state": "[]"
            },
            "attributes": {}
        }
    ],
    "resource": {
        "telemetry.sdk.language": "python",
        "telemetry.sdk.name": "opentelemetry",
        "telemetry.sdk.version": "1.12.0rc1",
        "telemetry.auto.version": "0.31b0",
        "service.name": "unknown_service"
    }
}
{
    "name": "/rolldice",
    "context": {
        "trace_id": "0xcebe72a8222da32d558132e0cd0546d0",
        "span_id": "0x7c0657cf75ba73f1",
        "trace_state": "[]"
    },
    "kind": "SpanKind.SERVER",
    "parent_id": null,
    "start_time": "2022-08-05T14:51:38.064308Z",
    "end_time": "2022-08-05T14:51:38.065943Z",
    "status": {
        "status_code": "UNSET"
    },
    "attributes": {
        "http.method": "GET",
        "http.server_name": "127.0.0.1",
        "http.scheme": "http",
        "net.host.port": 5000,
        "http.host": "127.0.0.1:5000",
        "http.target": "/rolldice",
        "net.peer.ip": "127.0.0.1",
        "http.user_agent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/103.0.0.0 Safari/537.36",
        "net.peer.port": 57130,
        "http.flavor": "1.1",
        "http.route": "/rolldice",
        "http.status_code": 200
    },
    "events": [],
    "links": [],
    "resource": {
        "telemetry.sdk.language": "python",
        "telemetry.sdk.name": "opentelemetry",
        "telemetry.sdk.version": "1.12.0rc1",
        "telemetry.auto.version": "0.31b0",
        "service.name": "unknown_service"
    }
}
```

**Comment by cartermp** [comment]
@melbgirl is this still on your radar to complete?

**Comment by cartermp** [comment]
Will close due to inactivity. Let me know if you'd like to continue and we can re-open!

**Pull request closed without merging**